contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.vue

```diff
@@ -28,18 +30,22 @@
         />
       </div>
       <VListTileContent
-        class="description-col pa-2 grow"
+        class="description-col pa-2"
         :class="{
           'my-4': !isCompact,
           'my-2': isCompact,
         }"
       >
         <VListTileTitle data-test="title">
-          <h3
-            class="text-truncate notranslate"
-            :class="{'font-weight-regular': isCompact}"
-          >
-            {{ node.title }}
+          <h3 :class="{'font-weight-regular': isCompact}">
+            <VLayout row>
```

Comment:
Phrasing content is allowed in an `h3`, which does not include `div`. I would suggest setting the `h3` as a child of the layout